### PR TITLE
Css four column layout

### PR DESCRIPTION
**Pull request by aklepper20**
css-four-layout-column | klepper

image

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.header {
+  font-weight: bold;
+  text-align: center;
+  font-size: 12px;
+  margin-bottom: -1.5%;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:34]: Negative values for margin are a no-no. Try to discover what element has margin or padding that you wanted to reduce here, and just change that measurement to a smaller value.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my note.

File: css-four-column-layout/styles.css (hunk 1 above)
Comment (aklepper20) [reply css-four-column-layout/styles.css]: image
I took out margin-bottom from the div the header lives in

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.container {
+  max-width: 700px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.header {
+  font-weight: ;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:34]: Empty property here, this can likely be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Very nice! One minor fix:

File: css-four-column-layout/styles.css (hunk 2 above)
Comment (aklepper20) [reply css-four-column-layout/styles.css]: image
Thank you!

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**